src/helpers/ExpenseHeadHelpef/expensehead.details.helper.ts

```diff
@@ -0,0 +1,147 @@
+import { BaseHelper } from '../BaseHelper/base.helper';
+import { expect } from '@playwright/test';
+
+import { NotesHelper } from '../BaseHelper/notes.helper';
+
+import { formatDate } from '@/utils/common.utils';
+
+export class ExpenseHeadDetailsHelper extends BaseHelper {
+    public noteHelper: NotesHelper;
+    public async init() {
+        await this.navigateTo('EXPENSE_HEADS');
+    }
+
+    public async clickOnExpenseHead(name: string) {
+        this._page.getByRole('tab', { name: 'All', exact: true }).click();
+
+        this._page.waitForTimeout(1000);
+
+        await this._page.getByText(name).click();
+
+        this._page.waitForTimeout(2000);
+    }
+
+    public async clickOnManagerName(name: string) {
+        await this._page.getByText(name).click();
+
+        this._page.waitForTimeout(2000);
+    }
+
+    public async clickOnEditIcon() {
+        await this._page.locator('.items-center > button').first().click();
+
+        this._page.waitForTimeout(2000);
+    }
+
+    public async EditExpenseHead(
```

Comment:
Please follow one standard. Function should always start with lower case alphabet.
```suggestion
    public async editExpenseHead(
```